### PR TITLE
pkp/pkp-lib#4984 Fixes the tasks popover

### PR DESCRIPTION
**Pull request by jonasraoni**
Added an option to clear the "callWhenClickOutside" callbacks and fixed the tasks popover.

Should I open a PR against OJS to update the /js/pkp.min.js?

### CODE REVIEW

File: js/pages/header/TasksHandler.js [hunk 1]
```diff
@@ -62,22 +62,27 @@
 	 */
 	$.pkp.pages.header.TasksHandler.prototype.appendToggleIndicator_ =
 			function(callingElement, event) {
-
 		var $header = this.getHtmlElement(),
-				$popover = $header.find('#notificationsPopover'),
-				$toggle = $header.find('#notificationsToggle');
+			$popover = $header.find('#notificationsPopover'),
```
Comment (asmecher) [inline js/pages/header/TasksHandler.js]: The JS linter will complain about this -- it expects a double-tab indent when continuing a previous line. Try `bash lib/pkp/tools/buildjs.sh -n` to run the linter locally.

Comment (asmecher) [reply js/pages/header/TasksHandler.js]: (Or you can create a PR that only updates the submodule with the specially-formulated commit name referring to your branch, i.e.: `pkp/pkp-lib#4984 Submodule update ##jonasraoni/i4984-fix-tasks-popover##` -- that'll cause Travis to run tests including a JS linting round. But that's a pretty painful way to work through JS linting errors.)

**Comment by asmecher** [comment]
Don't worry about recompiling the JS -- that's part of the build process, so we'll make sure we get to it before packaging up the next release.

File: js/pages/header/TasksHandler.js (hunk 1 above)
Comment (jonasraoni) [reply js/pages/header/TasksHandler.js]: @asmecher, I've updated the PR.
Just to confirm, it's expected that the `buildjs.sh` is emitting `1 error(s), 247 warning(s)`, right?

Comment (jonasraoni) [reply js/pages/header/TasksHandler.js]: I replaced the compiler.jar by the version 20120917 and looks like it fixed the problem =]

**Comment by jonasraoni** [comment]
@asmecher If there's nothing more missing, feel free to merge =]

**Comment by asmecher** [comment]
Thanks, @jonasraoni!

**Pull request merged**